lib/igniter.ex

```diff
@@ -333,16 +343,17 @@ defmodule Igniter do
   Finds the `Igniter.Mix.Task` task by name and composes it (calls its `igniter/2`) into the current igniter.
   If the task doesn't exist, a fallback implementation may be provided as the last argument.
   """
-  def compose_task(igniter, task, argv \\ [], fallback \\ nil)
+  def compose_task(igniter, task, argv \\ nil, fallback \\ nil)
```

Comment:
I think we might want to require that they provide the argv here still? What I worry about is accidentally making it easy to misuse the positional arguments to your task, that should very likely not be passed into a composed task.

Reply:
So what you're saying is that it's more likely that the correct thing here is to use `igniter.args.argv_flags`, but we should leave it up to the call site? So the pattern we should advertise in the docs will be:

```elixir
igniter
|> Igniter.compose_task("some.other.task", igniter.args.argv_flags)
```

Reply:
Yeah, although...now that you put it that way, maybe we should just make the default `igniter.args.argv_flags` 😆 then the default behavior is also the most likely correct behavior.

Reply:
👍 That sounds good to me. I'll update the documentation for `compose_task` to include that.

Reply:
Updated this in [0810f8b](https://github.com/ash-project/igniter/pull/131/commits/0810f8b649055f6df8d75e3b8534ced4fc60818c), please take a look at those changes and let me know your thoughts!